sdk-tests/src/test/java/io/dapr/it/testcontainers/DaprContainerTest.java

```diff
@@ -145,15 +140,20 @@ public void testPlacement() throws Exception {
 
   @Test
   public void testPubSub() throws Exception {
-    try (DaprClient client = (new DaprClientBuilder()).build()) {
-      client.waitForSidecar(1000).block();
+    DaprClientBuilder builder = createDaprClientBuilder();
 
+    try (DaprClient client = (builder).build()) {
       String message = "message content";
       Map<String, String> metadata = singletonMap(Metadata.TTL_IN_SECONDS, MESSAGE_TTL_IN_SECONDS);
       client.publishEvent(PUBSUB_NAME, PUBSUB_TOPIC_NAME, message, metadata).block();
     }
 
-    verify(getRequestedFor(urlMatching("/dapr/config")));
     verify(postRequestedFor(urlEqualTo("/events")).withHeader("Content-Type", equalTo("application/cloudevents+json")));
   }
+
+  private DaprClientBuilder createDaprClientBuilder() {
+    return  new DaprClientBuilder()
+        .withPropertyOverride(Properties.HTTP_PORT, String.valueOf(DAPR_CONTAINER.getHttpPort()))
```

Comment:
🔥🔥🔥🔥
I love the way that this worked out 👍

Reply:
Yeyyy